elephant/asset/asset.py

```diff
@@ -291,6 +294,8 @@ def _transactions(spiketrains, bin_size, t_start, t_stop, ids=None):
         (id, `neo.SpikeTrain`).
     """
 
+    logger.debug("Finding transactions")
+
```

Comment:
Shouldn't this be info?
```suggestion
    logger.info("Finding transactions")

```

Reply:
I remember adding this to be able to check if the extraction was working, as it can take a long time and one may suspect everything hung. But looking in more detail now, perhaps the best is to put `logger.info` calls in the main `extract_synchronous_events` method, as this may be called twice if `spiketrains_j` was passed, and the logging would be weird with two identical messages. This way, we could have explicit logging information messages for each list of spike trains being processed.

Reply:
Changed logging level to INFO and move logging call to `extract_synchronous_events` function, see: [3f5211a](https://github.com/NeuralEnsemble/elephant/pull/570/commits/3f5211a7a1c759ac20fa23b9b5c5f26e9d2d7fae)